Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/GenericComboBoxCtrl.inl

```diff
@@ -363,4 +431,6 @@ namespace AzToolsFramework
             genericGUI->m_postChangeNotifyCB->Invoke(notifyInstance, oldValue);
         }
     }
-}
+} // namespace AzToolsFramework
+#pragma optimize("", on)
+#pragma inline_depth()
```

Comment:
As kberg-amzn has mentioned, these pragmas need to be removed.

I believe the validator should catch these issues.